src/Core/Theme.php

```diff
@@ -34,6 +34,11 @@ public static function getAllowedList()
 		return $allowed_themes;
 	}
 
+	public static function setAllowedList(array $allowed_themes)
+	{
+		Config::set('system', 'allowed_themes', implode(',', $allowed_themes));
```

Comment:
Just stumbled over it, as github shows the two diffs alongside. With the addons, the implosion for the list is done with `, ` and with the themes it is only `,`. Wonder if it makes sense to only use one separation way for comma separated lists.

Reply:
In this case we would need to use the `,<space>` form to include both forms as a backward-compatibility feature.